spec/models/category_spec.rb

```diff
@@ -0,0 +1,6 @@
+require 'spec_helper'
+
+describe Category do 
+
```

Comment:
Try to remove unnecessary blank lines, they break up the flow of the code. That said, you should still use blank lines to break up sections of code. Variable assignments from other code, for example.